dev/io.openliberty.jcache.internal_fat.infinispan/publish/servers/io.openliberty.jcache.internal.fat.auth.cache.restart.1/bootstrap.properties

```
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=*=info:jcache=all:com.ibm.ws.security.*=all:io.openliberty.security.*=all
```

I added `com.ibm.websphere.security.*=all` to all the other servers. Should add it here as well.

```suggestion
com.ibm.ws.logging.trace.specification=*=info:jcache=all:com.ibm.ws.security.*=all:com.ibm.websphere.security.*=all:com.ibm.ws.webcontainer.security.*=all
```